SUBGROUPS.md

```diff
@@ -84,20 +84,18 @@
 * Status
   * Active
 
-## Events and Outreach
+## Outreach Committee
 
 * Description & Responsibilities
-  * The charter of the subgroup is to plan, coordinate, and execute DB
-    community and other industry events as well as lead outreach, marketing,
-    and PR activities for DB
+  * The charter of the subgroup is where recruiting for additional project members, events coordination, vision discussions, and marketing related activities will take place. sometimes referred to as the project's non-technical, non-governing board activities

```

Comment:
A few nits:
1. Prefer to wrap long lines
2. "sometimes" -> "Sometimes"
3. Second sentence is incomplete

Reply:
@seroyer fixed 1, 2, and 3.